### ISSUE TITLE
AssetLib Icon looks like a padlock

### ISSUE THREAD

**Issue by ghost** [godotengine/godot-design#17]
I've been getting some feedbacks on Twitter about how the AssetLib icon looks like a padlock.
This might make it more difficult for users (it took me a while too) to find the link for AssetLib.
What do you think? :)

https://twitter.com/NebulaBytes/status/976096307629588480

**Comment by akien-mga** [comment]
I agree - and since it's a *library* and not a *store*, it would be better to make a reference to something library-ish than a shopping bag :)

**Comment by djrm** [comment]
Ok, will try to find something more understandable

**Comment by djrm** [comment]
Ok here are some options for the new asset lib icon, personally i prefer the option 3, since even if the asset lib is not a paid store, the icons represents a basket, which is an icon generally used for marketplaces, so people know what it means (assuming they undestand common icons, otherwise there is not much to do) even if they discover that everything is free once they get there, also creating an icon that conveys the meaning of freedom and asset library would be quite hard.

For the option 2, it is nice, however, in the website could be easily confused with a journal or a manual, so it is not the best option, even though the meaning in editor is library.

the option 1 is an object library (assuming godot engine's icon language), also not the best option for an asset library.

Also if the problem is the webpage, i can remove the icon and put text instead, so people dont get confused.

![screenshot from 2018-04-28 18-19-15](https://user-images.githubusercontent.com/1103897/39401682-b54a273e-4b10-11e8-9d59-80875eb2d67e.png)

**Comment by bojidar-bg** [comment]
3 looks best from the ones above, the other two are going to be confusing.

**Comment by pwab** [comment]
> Also if the problem is the webpage, i can remove the icon and put text instead, so people dont get confused.

I would prefer this solution over using icons. It could look like this:
![assets](https://user-images.githubusercontent.com/19271185/39410520-c473d55e-4bf9-11e8-9d4d-a6bf3af25ad2.png)

Using an icon is confusing because it is the only one in the nav bar right now. It would make more sense if every link has its own icon. So instead of adding a handful of icons I would vote for removing this one.

**Comment by nobuyukinyuu** [comment]
The editor icon still reminds me somewhat of a stylized lock or [anchor weight](https://user-images.githubusercontent.com/1023003/60632902-b7432c00-9dcd-11e9-8ff5-7b462271c205.png).   I happen to like the book part in option 1, just by itself.  How about an open book perhaps?

![image](https://user-images.githubusercontent.com/1023003/60633729-9b418980-9dd1-11e9-8413-d7605a68fef0.png)


Same license as this repo: 
[icon_asset_lib.zip](https://github.com/godotengine/godot-design/files/3357282/icon_asset_lib.zip)
(Sorry for the closed-source format;  will upload an SVG on request but might lose layers and un-expanded lines.  Perhaps this will open in inkscape)

**Comment by akien-mga** [comment]
The high res version looks nice, but the small icon (which is the one people will see) is maybe not so readable. Keep in mind when designing that you are design for this small icon size, and it should as well as it can. It looks fine when you know how the big icon looks like, but if you've never seen it big, understanding the small icon may not be straightforward.

As for using a book, it looks nice but I fear people would confuse it for the documentation.

**Comment by FeatherAntennae** [comment]
The asset lib is about downloading assets, so why not a standard download icon like this?

![Download](https://www.iconfinder.com/icons/2561424/download_icon)
